lib/taskana-core/src/test/java/acceptance/classification/CreateClassificationAccTest.java

```diff
@@ -1,297 +1,374 @@
+    String invalidCategory = "UNKNOWN_CATEGORY";
+
     Classification classification =
-        CLASSIFICATION_SERVICE.newClassification("Key3", "DOMAIN_A", "UNKNOWN_TYPE");
-    assertThatThrownBy(() -> CLASSIFICATION_SERVICE.createClassification(classification))
-        .isInstanceOf(InvalidArgumentException.class);
+        classificationService.newClassification("KeyErrCreation", "DOMAIN_A", "TASK");
+    classification.setCategory(invalidCategory);
+
+    assertThatThrownBy(() -> classificationService.createClassification(classification))
+        .isInstanceOf(InvalidArgumentException.class)
+        .hasMessage(
+            String.format(
+                "Given classification category %s with type TASK is not valid according to the"
+                    + " configuration.",
+                invalidCategory));
   }
 
   @WithAccessId(user = "businessadmin")
   @Test
-  void testCreateClassificationOfUnknownCategory() {
+  void should_ThrowException_TryingToCreateClassificationWithInvalidParentKey() {
+    String invalidParentKey = "UNKNOWN_KEY";
+
     Classification classification =
-        CLASSIFICATION_SERVICE.newClassification("Key4", "DOMAIN_A", "TASK");
-    classification.setCategory("UNKNOWN_CATEGORY");
-    assertThatThrownBy(() -> CLASSIFICATION_SERVICE.createClassification(classification))
-        .isInstanceOf(InvalidArgumentException.class);
+        classificationService.newClassification("KeyErrCreation", "", "TASK");
+    classification.setParentKey(invalidParentKey);
+
+    // Make sure that there is no (Parent) Classification with this Key.
+    List<ClassificationSummary> classifications =
+        classificationService.createClassificationQuery().keyIn(invalidParentKey).list();
+    assertThat(classifications).isEmpty();
+    assertThatThrownBy(() -> classificationService.createClassification(classification))
+        .isInstanceOf(InvalidArgumentException.class)
+        .hasMessage("Parent classification could not be found.");
   }
 
   @WithAccessId(user = "businessadmin")
   @Test
-  void testCreateClassificationWithInvalidParentId() {
-    Classification classification = CLASSIFICATION_SERVICE.newClassification("Key5", "", "TASK");
-    classification.setParentId("ID WHICH CANT BE FOUND");
-    assertThatThrownBy(() -> CLASSIFICATION_SERVICE.createClassification(classification))
-        .isInstanceOf(InvalidArgumentException.class);
+  void should_ThrowException_TryingToCreateClassificationWithInvalidParentId() {
+    String invalidParentId = "UNKNOWN_ID";
+
+    Classification classification = classificationService.newClassification("KeyErr", "", "TASK");
+    classification.setParentId(invalidParentId);
+
+    // Make sure that there is no (Parent) Classification with this Id.
+    List<ClassificationSummary> classifications =
+        classificationService.createClassificationQuery().idIn(invalidParentId).list();
+    assertThat(classifications).isEmpty();
```

Comment:
remove. Not necessary

Reply:
yep, for id I can see thats unnecessary. Thanks ^^